pallets/exchange-rate/Cargo.toml

```diff
@@ -32,55 +32,46 @@ serde = { version = '1.0.101', optional = true, features = ['derive'] }
 [dependencies.pallet-balances]
 default-features = false
 git = 'https://github.com/paritytech/substrate.git'
-branch = "rococo-v1"
-version = '3.0.0'
+branch = "polkadot-v0.9.9"
```

Comment:
is there are reason why we didn't update to `polkadot-v0.9.10` like in latest substrate-parachain-template?

Reply:
noted that this will be future PR